includes/wikia/services/AvatarService.class.php

```diff
@@ -269,7 +269,7 @@ public static function getVignetteUrl( Masthead $masthead, $width, $timestamp )
 				$url = self::buildVignetteUrl( $width, $bucket, $relativePath, $timestamp, false );
 			}
 		} else { // default avatar
-			$legacyDefaultUrl = array_shift( $masthead->getDefaultAvatars( 'thumb/' ) );
+			$legacyDefaultUrl = @array_shift( $masthead->getDefaultAvatars( 'thumb/' ) );
```

Comment:
Right, that was a quick fix as strict warnings were driving me crazy :)

Sure, needs to be properly fixed. Storing the function result in a temporary variable that will be passed to `array_shift`.